packages/@aws-cdk/aws-eks/kubectl-layer/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# base lambda image
+FROM public.ecr.aws/lambda/provided:latest
+
+# versions
+ARG KUBECTL_VERSION=1.20.0
+ARG HELM_VERSION=3.4.2
+ARG AWSCLI_VERSION=1.18.198
+
+USER root
+RUN mkdir -p /opt
+WORKDIR /tmp
+
+# install some tools
+RUN yum update -y \
+    && yum install -y zip unzip make wget tar gzip
```

Comment:
For all software we bundle, I believe we should need them to add them to the NOTICE file. 

We've done so in other places where the bundling occurs via npm - https://github.com/aws/aws-cdk/blob/d5afb555b983c8c034f63dd58d1fa24b82b6e9fe/tools/pkglint/lib/rules.ts#L141

Reply:
Thanks, let me have a look

Reply:
Okay, added attribution